tests/views/pages/forgot.handlebars

```diff
@@ -0,0 +1,41 @@
+<html>
+    <head>
+        <title>"Forgot Password" Form (used for client-side tests)</title>
+
+        {{>common-includes}}
+
+        <!-- Bring in client library being tested -->
+        <script type="text/javascript" src="/js/client/forgot.js"></script>
+    </head>
+    <body>
+        <!-- Sample "forgot password" form with all required markup -->
+        <div class="row">
+            <div class="column small-offset-4 small-4 forgot-viewport">
+                <div class="forgot-message">
+                    <p>Please enter the email address you used when creating your account, and a password reset email will be sent to you.</p>
+                </div>
+                {{>forgot-form}}
+            </div>
+        </div>
+
+        <!-- configure and instantiate our module -->
+        <script type="text/javascript">
+            var forgot = gpii.express.couchuser.frontend.forgot("body", {
```

Comment:
Try to get as much JavaScript out of the templates as possible. For a start, it seems that all the templateUrls are equal to the default and can be removed - also, I believe you don't store in them in the "model" any more anyway. It would be great if the handlebars helper could be improved so that these "init blocks" could be stored as JSON or JS files (and they could then be linted) and woven into the base of the document automatically.